cookiecutter.json

```
@@ -0,0 +1,10 @@
+{
+    "date": "2019.06.01",
```

I think the date can be generated using jinja
```suggestion
    "date": "{% now 'utc', '%Y.%m.%d' %}",
```

---

Actually, if we can access the date through jinja, they probably don't need to be variables? Just use jinja directly in the file where you need the value.

---

Jinja could be used directly in the file